### PR TITLE
simdjson 2.2.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by carlocab** [comment]
For future reference, please put the name of the formula modified first in your commit message header, like so:

```
quick-lint-js: bump revision (simdjson 2.2.0)
```

If you use
```
brew bump-revision --message "(simdjson 2.2.0)" quick-lint-ha
```
then `brew` handles the rev bump and commit for you. (You still need to do `git push`.)

**Comment by carlocab** [comment]
`simdjson` adjusted `SIMDJSON_PADDING` in https://github.com/simdjson/simdjson/pull/1856 from `32` to `64`, which causes the [`static_assert` in `quick-lint-js`](https://github.com/quick-lint/quick-lint-js/blob/f5301aaf483d79514167ef518e7bb9526d5f2fa3/src/padded-string.cpp#L16-L17) to fail.

File: Formula/quick-lint-js.rb [hunk 1]
```diff
@@ -34,6 +34,9 @@ class QuickLintJs < Formula
   fails_with gcc: "6"
   fails_with gcc: "7"
 
+  # Fix build with `simdjson 2.2.0`
+  patch :DATA
```
Comment (carlocab) [inline Formula/quick-lint-js.rb:38]: Will upstream this if it works.

Comment (carlocab) [reply Formula/quick-lint-js.rb]: https://github.com/quick-lint/quick-lint-js/pull/779

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**